cli/mypy.ini

```diff
@@ -16,3 +16,9 @@ warn_redundant_casts = True
 warn_return_any = True
 warn_unused_configs = True
 warn_unused_ignores = True
+
+[mypy-azure.storage.blob.*]
+; starting in azure 12.9.0:
+; site-packages/azure/storage/blob/_serialize.py:129: error: Type signature has too many arguments
+; site-packages/azure/storage/blob/_blob_client.py:1406: error: Type signature has too few arguments
+follow_imports = skip
```

Comment:
why not `error`/`silent`? skip seems like the most extreme.

Reply:
Good question.  I hadn't actually looked at the options before, I just copied from above.

But I just checked, and since this is somebody else's code... `error` just prints noise over code we can't control, and `silent` actually still fails afterwards.